data/campaigns/The_South_Guard/scenarios/02_Proven_by_the_Sword.cfg

```diff
@@ -549,24 +555,43 @@
         [/kill]
 
         [if]
-            [have_unit]
-                id=Minister Hylas
-                x,y=$x2,$y2
-            [/have_unit]
-
+            [not]
+                [have_unit]
+                    id="Minister Hylas"
+                [/have_unit]
+            [/not]
             [then]
-                [message]
-                    speaker=Deoran
-                    message= _ "How can a bandit have summoned undead?"
-                [/message]
+                {MINISTER_HYLAS}
```

Comment:
Also there for victory before finding Hylas. And also probably going away because it's best done on the way out the door than littered throughout the code like it was before I started using session_end. (Yes, will roll it all onto the end of victory, instead, as you asked)